### ISSUE TITLE
Grouping does not work in swagger ui 2.1.4 and grape-swagger 0.20

### ISSUE THREAD

**Issue by stefan-kolb** [stefan-kolb/nucleus#72]
![image](https://cloud.githubusercontent.com/assets/2141507/14851257/0887d8ce-0c7f-11e6-96c2-16ae72d0277b.png)
![image](https://cloud.githubusercontent.com/assets/2141507/14851264/0f04911a-0c7f-11e6-992d-81f26e9e21c7.png)

**Comment by croeck** [comment]
Could be somehow related to a recent comment which was made on one of the changes I made in `grape-swagger` to enable this feature. I am not 100% sure if this is related to `grape-swagger`, but I would assume so.
My original code snippet is still enabled and the documentation also describes this feature, but all my tests were not migrated to version `0.20` wherefore I am not convinced this is still working.

**Comment by stefan-kolb** [comment]
Good point. I saw that they migrated the code and documentation, but didn't check the tests yet. Seems like they are moving a bit fast and break too much things at the moment :cry: I mean it's not absolutely necessary but still was a nice feature.

**Comment by stefan-kolb** [comment]
See https://github.com/ruby-grape/grape-swagger/issues/414
